src/game.js

```diff
@@ -0,0 +1,20 @@
+class Game {
+    constructor() {
+        this.rolls = [];
+    }
+
+    Game; roll = function (pins) {
+        this.rolls.push(pins);
+    };
+
+    Game; score = function () {
+        var result = 0
+        var rollIndex = 0;
+
+        for (var frameIndex = 0; frameIndex < 10; frameIndex++) {   //frameIndex go through 10 frame
```

Comment:
Nice clear logic, this is a solid start for expanding to spare and strike scoring